packages/core/src/components/overflow-list/overflow-list.md

```diff
@@ -1,6 +1,19 @@
 @# Overflow list
 
-`OverflowList` displays a list of items that can overflow on its own. The required `visibleItemRenderer` callback prop allows for customizing the appearance of visible items. The required `overflowRenderer` callback prop allows for customizing the rendering of collapsed items. 
+`OverflowList` takes a generic list of items and renders as many items as can
+fit inside itself. Other items are collapsed into an overflow menu. The visible
```

Comment:
> Other items are collapsed into an overflow menu.

That's quite opinionated, I preferred the more generic previous wording. Can you replace `overflow menu` with `overflow element`?